index.html

```diff
@@ -1,12 +1,41 @@
-Shu Hu 
-About
-Hi! I am a current master's student in humanities and I used to be a PR/communications professional in tech industry.
-Experience
-Volkswagen Mobvoi, product launch planning/execution project
-Tencent, digital summit planning/organization project
-BlueFocus Marketing Agency, product launch/CEO event planning/execution project
-Rare Media, international documentary shooting project 
-Connect
-Email: shuhu2023@gmail.com
-Github: https://github.com/dreamlala2023
-D:\codingfiles\freyr-intro\instructions\section-2\index.html
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Shu Hu's profile</title> 
```

Comment:
I will approve these changes because everything else looks good. One comment is that you should add some meta elements which is the third checkpoint for the Head section:

![image](https://github.com/dreamlala2023/freyr-intro/assets/43383899/6eeb5c2e-47aa-44f6-9ec6-05f9cabf9e16)